apps/docs/pages/styled-system/outline-width.mdx

```diff
@@ -9,7 +9,10 @@ Use `outline` prop to control the outline width of an element.
 
 ## Usage
 
-<ax-demo name="outline-width/usage" />
+<ax-demo
+  meta='/outline="1"/ /outline="2"/ /outline="4"/'
```

Comment:
allow highlighting words in code snippets

<img width="864" alt="Screenshot 2024-05-21 at 3 40 15 PM" src="https://github.com/optimizely-axiom/optiaxiom/assets/758244/fdd12fec-abad-4341-a037-54ad2d4c2e67">